portfolio/src/main/webapp/resume.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <link href="style.css" rel="stylesheet">
+        <title>Victor's Resume</title>
+        <style>
+            h2 {
+                margin-left: 20px;
+                text-decoration: underline;
+                font-family: "Times New Roman";
+                font-size: 26px;
+            }
+            p {
+                font-family: "Merriweather";
+                font-size: 18px; 
+                margin-left: 30px;
+            }
```

Comment:
I would add these styles to a css file. You have two options here:

1) Create a resume.css file and copy these in directly, and then on line 5, instead of referencing style.css, you can reference resume.css. That way these styles won't bleed into other pages that use style.css (unless you want all h2 and p tags to look like this throughout your website, in which case you can put them into style.css, option 2). Note that if you do this, you won't get the styles from style.css, so some things might break
2) Throw these styles into style.css, and h2 and p tags will be updated for every file that uses style.css as your stylesheet

Up to you which option you choose, depending on how you want your site to behave